### PR TITLE
Thingweb website redesign

### PR DESCRIPTION
**Pull request by SergioCasCeb**
Website preview: [Thingweb website](https://thingweb.netlify.app)

Everything in this PR:

* Overall new design for the website utilizing Docusaurus
* New landing page
  * Hero banner
  * Value proposition section
  * Worklfows section (still under construction)
  * Tools section
  * Services section
  * Community section (Adopters list)
* Docs page for documentation
* Services page and hands-on page (currently just a copy of the old one)

Also coming in this PR:
* Google Analytics banner
* Feeback prompt/icon

### CODE REVIEW

File: misc/designcrowd/fonts.zip
Comment (egekorkan) [inline misc/designcrowd/fonts.zip:1]: @danielpeintner this should not be needed anymore right? It is mentioned at https://github.com/eclipse-thingweb/website/pull/67#issuecomment-1911811144 but it is old

Comment (danielpeintner) [reply misc/designcrowd/fonts.zip:1]: I agree, it should not be needed anymore. Does it make sens to store them "somewhere".. or is the GitHub archive enough?

**Review by danielpeintner — CHANGES_REQUESTED** [review]
Find some more comments below... based on the the rendered version at https://thingweb.netlify.app/

## Intendation in code snippets broken.
e.g.,, https://thingweb.netlify.app/hands-on/intro-raspberry/

image

There are more places...

## Spaces in Services missing
image

There are more..

## Navigation sub navi
e.g. in Hands-on for example etc

## Shall we open links in new window/tab or not?
We should be consistent..
for example  Adopters don't open in new page.. playground does.. not sure what we can do. Do know that Adapteres is code from Eclipse

## TD specification link

At the moment it points to dated URI, https://www.w3.org/TR/2023/REC-wot-thing-description11-20231205/
All the other don't do that.. I think we should use https://www.w3.org/TR/wot-thing-description11/ instead...

## Hyphens in tools
Tools section.. Sometimes with dash between name, sometimes not 
dart-wot vs dart_wot, playground without hyphen at all? Maybe also without TD ?

## Draggable moving parts
Would be nice if we find a good way to drag manually the sliders...


## General

I am okay with merging soon and fixing issues afterwards and once the popup...

File: CONTRIBUTING.md
[the remark this answers is not in the document]
Comment (egekorkan) [reply CONTRIBUTING.md:1]: This will be restored

**Comment by egekorkan** [comment]
@danielpeintner your feedback is now implemented and deployed.

**Comment by egekorkan** [comment]
Descriptions for tools can be picked from https://github.com/eclipse-thingweb/.eclipsefdn/pull/29

File: src/components/FeedbackPrompt/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export default function FeedbackPrompt() {
+
+    return (
+        <div className={styles.feedbackPrompt}>
+            <div className={styles.feedbackPrompt__text}>
+                <p className={styles.text1}>Liked or disliked the new website?</p>
+                <p className={styles.text2}>Leave us some feedback!</p>
+            </div>
+            <div className={styles.feedbackPrompt__btn}>
+                <a href='https://github.com/eclipse-thingweb/website' target='_blank'>
```
Comment (egekorkan) [inline src/components/FeedbackPrompt/index.js:17]:
```suggestion
                <a href='https://github.com/orgs/eclipse-thingweb/discussions/8'>
```

**Comment by danielpeintner** [comment]
> @danielpeintner your feedback is now implemented and deployed.
> 
> Regarding hyphens: We have reached a compromised resolution that we are happy with, which is removing the hyphen and putting a space instead.

Thanks 👍

In some code snippets the code seems still a bit odd and inconsistent (different indentation for examples that follow each other etc). Maybe it was there before on the old website. If there would be an autoformatter it might be even better ... anyhow.. I think it is fine as is!

BTW. the "leave us some feedback" banner takes quite a bit of space on smaller screen sizes and hides other content? Do we actually need that at all?
Anyhow, again no really strong opinion.

File: src/components/HomogeneousWorkflow/index.js
Comment (egekorkan) [inline src/components/HomogeneousWorkflow/index.js:1]: I think we can change this to "SDK". Then we can merge and find a better name later on.

File: src/components/WorkflowsSection/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                            </div>
+                        </div>
+
+                        <div className={clsx(styles.workflow, styles.professionalW)} id='workflow' data-workflow="simple">
+                            <div className={styles.workflow__illustration}>
+
+                            </div>
+                            <div className={styles.workflow__content}>
+                                <h1 className={styles.title}>Orchestration</h1>
+                                <p className={styles.description}>Easily manage a fleet of heterogeneous devices and orchestrate applications at scale.</p>
```
Comment (egekorkan) [inline src/components/WorkflowsSection/index.js:98]: Can be called "Platform"

**Comment by egekorkan** [comment]
This is now good to go. I will resolve conflicts and merge

**Pull request merged**